lib/routes/account.js

```diff
@@ -536,6 +536,11 @@ module.exports = function (
           response.keyFetchToken = keyFetchToken.data.toString('hex')
 
           if (doSigninConfirmation) {
+            log.activityEvent('account.signin.verify.start', request, {
```

Comment:
Also, let's see if we can keep terminology consistent with "confirm" rather than "verify" to reduce future potential for confusion :-)